src/screening/eid/EidSummary.jsx

```diff
@@ -1,13 +1,10 @@
 import React from "react";
-import {ObsHistory} from '@openmrs/react-components';
-import PatientLabTests from "../../patient/PatientLabTests";
-import {CONCEPTS} from "../../constants";
+import { ObsHistory } from '@openmrs/react-components';
+import { CONCEPTS } from "../../constants";
 
 const EidSummary = props => {
   return (
     <span>
-      <PatientLabTests test_type={ "viral_load_tests" }/>
-      <PatientLabTests test_type={ "hiv_tests" }/>
```

Comment:
@jomadoye  is this related to this pull request, or for something else?

Reply:
oh, I mistakenly deleted that, updated.